data/hai/hai reveal 6 end.txt

```diff
@@ -0,0 +1,673 @@
+			choice
+				`	"Who is 'us'?"`
+			`	She sighs, "My father and I, and a couple of the old survivors of the purge. It's only me left now though, just one Base Principle."`
+			`	You shake your head and ask, "Do you realise you nearly started a war that would have killed millions? Possibly ended human government entirely!"`
+			`	"I know," she says, calm now, "obviously not my intent, but maybe that would have been better in the long-run than the Republic and its corruption. There's no apology appropriate for it, and I can't pretend it would be totally sincere on that front. War changes things, sometimes some pain in the short term is needed for a better future."`
+			`	"The Free Worlds were almost the perfect catalyst, but none of us could have predicted the Pug. Still, that nearly forced Alastair into retirement, it was almost enough. Were it not for the Pug I would already be running the Syndicate, and there would have been no further risk with the Hai. None of this would have ever happened."`
+			`	Her expression is defiant, "No one would have cared about a few missing people in the world I would have built! A necessary sacrifice in the name of progress, and I would've shut down the pirates myself."`
+			choice
+				`	"It's always fine to say as such about plans that didn't happen."`
+				`	"I think people would always still have had a problem with kidnappings."`
+			`	She gives you a hard stare, but her resolve isn't up to staring down a war hero from her knees, she looks away and you can see her blinking away more tears.`
+			`	"Is this to be my end then?" she aks, "Is this the price I pay for my efforts?"`
+			label decide
+			choice
+				`	(Spare her.)`
+					goto spare
+				`	(End her.)`
+					goto end
+				`	"Am I to be judge jury and executioner?"`
+					goto judge
+			label spare
+			`	You draw your weapon and point it at her, your thumb on the safety. When you look inside yourself though, you think maybe the future is worth the sacrifice made in the attempt.`
+			apply
+				set "spared terry"
+			`	The holster clicks as you put the weapon away.`
+				goto return
+			label end
+			`	You draw your weapon and point it at her, your thumb on the safety. Some prices are too high to pay for a better future.`
+			apply
+				set "ended terry"
+			`	The retort of the weapon is taken away on the wind.`
+				goto return
+			label judge
+			`	"You'd better be," she says, "you and I both know that this can't become public. If you take me to the Syndicate they will do away with me if they find out, and I doubt they could manage to not."`
+			`	"Oh, Alastair did tell me to say you're fired if you don't get back quickly... that was over a week ago now," you say.`
+			`	She rolls her eyes, "Of course, this is exactly the sort of callous governance I expect of them."`
```

Comment:
Small continuity question, it seems like the only person who knows anything about Terry's involvement with the base principles is the player character. Additionally, the loss of Philosopher's Paradise to her plans is devastating but her position within the syndicate seems to not have been jeopardized. This may be an unwanted suggestion (not as if my previous comments were wanted) but the story does seem to leave open the possibility of the player character joining forces with Terry against the Syndicate. idk its all praxis to me.